js/samples/05.chatModeration/teamsapp.local.yml

```diff
@@ -69,23 +69,6 @@ provision:
 # Defines what the `deploy` lifecycle step does with Teams Toolkit.
 # Runs after `provision` during Start Debugging (F5) or run manually using `teamsfx deploy --env local`.
 deploy:
-  # Ensures that the project dependencies are installed.
-  # - uses: cli/runNpmCommand
-    # with:
-    #   args: install --no-audit
-  - uses: script
-    with:
-      run: yarn
```

Comment:
Nice to have: we may still need this yarn command to install dependencies for people who only want to try the sample. So developers can navigate to this folder, provide the openai key and press F5 to try the sample directly.

I see there is step in readme to run yarn under the `js` folder, which installs dependencies for all the sub folders. I'm not sure whether we want to enable another way to try the sample without installing dependencies for all the js projects, as the sample is quite standalone.

Reply:
Yup, great point Chaoyi. Kavin and I have been discussing this and right now, our standard is that we want the samples to consume the local package so the install has to happen in the root js folder. However, we may explore allowing the build in the sample folders. There are tradeoffs to both, especially with dependencies/blockers that we'll need to consider.